PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditResult.kt

```diff
@@ -1,22 +1,29 @@
 package com.braintreepayments.api.paypal.vaultedit
 
 import com.braintreepayments.api.ExperimentalBetaApi
+import com.braintreepayments.api.core.PostalAddress
 
 @ExperimentalBetaApi
 sealed class PayPalVaultEditResult {
 
     /**
      * The PayPal vault edit flow completed successfully.
      */
-    @Suppress("EmptyDefaultConstructor")
-    class Success(
-        // TODO: add FI details
+    class Success internal constructor(
```

Comment:
Should we have doc strings for these params?

Reply:
Ah yeah we should. Let me update this.